Java/measureeval/src/main/resources/logback-spring.xml

```diff
@@ -41,17 +42,20 @@
         </then>
     </if>
 
+    <logger name="org.opencds.cqf.cql.engine.debug.DebugUtilities" level="DEBUG" additivity="false">
+        <appender-ref ref="cql"/>
+    </logger>
+    <logger name="com.lantanagroup.link.measureeval.services" level="WARN"/>
```

Comment:
Tangential, but do we have a project convention on logging levels?  I gravitate toward using a root level of `INFO` and then cranking down the entirety of `com.lantanagroup.link` to `DEBUG`.  (Regardless, if we set the root level to `WARN`, there's no need to specify `WARN` for individual loggers.)

Reply:
Agreed. Changed root level to INFO and link-specific classes to WARN.